src/server/game/Spells/SpellMgr.cpp

```diff
@@ -2821,6 +2822,10 @@ void SpellMgr::LoadSpellInfoCustomAttributes()
                 case SPELL_AURA_POWER_BURN:
                     spellInfo->AttributesCu |= SPELL_ATTR0_CU_NO_INITIAL_THREAT;
                     break;
+                case SPELL_AURA_PERIODIC_DAMAGE:        // DoTs like Shadow Word: Pain
+                    spellInfo->AttributesCu |= SPELL_ATTR0_CU_NO_INITIAL_THREAT;
```

Comment:
Please look at this line on master:

https://github.com/TrinityCore/TrinityCore/blob/3.3.5/src/server/game/Spells/SpellMgr.cpp#L2813

In my commit I only moved the "case SPELL_AURA_PERIODIC_DAMAGE:" so I would be able to add the SPELL_ATTR0_CU_AURA_CC flag.